app/v1/models/domain/StatusReasonEnum.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.domain
+
+import play.api.libs.json.Format
+import utils.enums.Enums
+
+sealed trait StatusReasonEnum {
+  def toDesViewString: String
+}
+
+object StatusReasonEnum {
+
+  case object signUpReturnAvailable extends StatusReasonEnum {
+    override def toDesViewString: String = "Sign up - return available"
+  }
+
+  case object signUpNoReturnAvailable extends StatusReasonEnum {
+    override def toDesViewString: String = "Sign up - no return available"
```

Comment:
really minor thing, but these don't need the ": String" types; Scala compiler tends to catch on with string & primitive literals

Reply:
I get a weak warning when I removed the type annotations